test/AnnotationProcessor/EnumItemInformationTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+namespace Hostnet\Component\AccessorGenerator\AnnotationProcessor;
+
+use Doctrine\Common\Util\Inflector;
+
+class EnumItemInformationTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * This is array.
+     */
+    public const A_TEST_CONSTANT = 'I_TEST_CONSTANT';
```

Comment:
what bad English, I thought.... 
'This is a constant representing an array'
etc